### PR TITLE
changed spaced indentation to tabs

### PR DESCRIPTION
**Pull request by Salvakiya**
This is more compatible with various Godot versions. 3.2.3 changes 4 spaces to tabs by default when you run the project. This means that you get mixed indentation if you use 2 space indentation.

### CODE REVIEW

**Comment by clemens-tolboom** [comment]
I cannot run the example for same reasons.

This PR looks good to me :-)

File: addons/GDCharts/script.gd [hunk 1]
```diff
@@ -91,532 +91,531 @@ onready var global_scale = Vector2(1.0, 1.0) / sqrt(MAX_VALUES)
...
-    if abs(amplitude) > computed_ratio:
-      ratio = computed_ratio
-      ordinate_values = []
+		if abs(amplitude) > computed_ratio:
+			ratio = computed_ratio
+			ordinate_values = []
 
-      for index in range(-6, 6):
-        ordinate_values.push_back(5 * index * computed_ratio / ordinary_factor)
+			for index2 in range(-6, 6):
```
Comment (clemens-tolboom) [inline addons/GDCharts/script.gd:613]: This is changed to sub_index in master branch

**Comment by clemens-tolboom** [comment]
Why is this closed without a reason? No credits for @Salvakiya :-/

I neither see a commit fixing this PR apart from my comment above in d30c57628e016877e51833c3c77836391f4d0802

@xsellier this project does not run in (my) Godot due to space mixed with tabs which IMHO opinion was the intend of @Salvakiya right?

**Comment by xsellier** [comment]
Hey,

So, I use spaces not tabulations in my projects, Im not going to change that.
Thanks to your IDE you should be able to easily change that, but I won't update that repository for tabulation instead of spaces.

Sorry.

**Comment by xsellier** [comment]
More in depth explanations:
- This PR contains conflicts
- No update for 2 months
- Updated only the script.gd file, but not the plugin.gd or any GDScript example files.

This project misses a `CONTRIBUTE.md` file indeed.
There is another gd-chart project which is really complete, and I suggest you to use it instead:
https://github.com/fenix-hub/godot-engine.easy-charts

**Comment by clemens-tolboom** [comment]
So I do:
- git clone git@github.com:binogure-studio/chart-gd.git
- start Godot `3.3.2.stable` and get immediately
> Unable to load addon script from path: 'res://addons/GDCharts/plugin.gd'. This might be due to a code error in that script.
Disabling the addon at 'res://addons/GDCharts/plugin.cfg' to prevent further errors.

- Ignoring that I can run the example scene :-)
- Not sure what went wrong the previous time triggering my `tabs` versus `spaces` :-/

**Comment by xsellier** [comment]
I suppose you can do that directly in godot, change the Text Editor > Indent > Type setting in the Editor Settings to default to space

**Comment by xsellier** [comment]
image

**Comment by clemens-tolboom** [comment]
Hmmm ... found tabs in example.gd :-p

My console output has some more

> ERROR: _load_data: Condition "!f" is true. Returned: ERR_CANT_OPEN
   At: scene/resources/texture.cpp:502.
ERROR: _load: Failed loading resource: res://.import/graph-plot-white.png-b2db87f2374c8f05649a10994a5a9092.stex. Make sure resources have been imported by opening the project in the editor at least once.
   At: core/io/resource_loader.cpp:283.
ERROR: _load: Failed loading resource: res://addons/GDCharts/graph-plot-white.png. Make sure resources have been imported by opening the project in the editor at least once.
   At: core/io/resource_loader.cpp:283.
SCRIPT ERROR: GDScript::reload: Parse Error: Can't preload resource at path: res://addons/GDCharts/graph-plot-white.png
   At: res://addons/GDCharts/script.gd:21.
ERROR: reload: Method failed. Returning: ERR_PARSE_ERROR
   At: modules/gdscript/gdscript.cpp:583.
SCRIPT ERROR: GDScript::reload: Parse Error: Couldn't fully preload the script, possible cyclic reference or compilation error. Use "load()" instead if a cyclic reference is intended.
   At: res://addons/GDCharts/plugin.gd:6.
ERROR: reload: Method failed. Returning: ERR_PARSE_ERROR
   At: modules/gdscript/gdscript.cpp:583.
ERROR: set_path: Another resource is loaded from path 'res://addons/GDCharts/graph-plot-white.png' (possible cyclic resource inclusion).
   At: core/resource.cpp:82.

**Comment by clemens-tolboom** [comment]
I did not know `Convert indent on Save` :-)

(case closed)

**Pull request closed without merging**